clang/test/Analysis/bstring_UninitRead.c

```diff
@@ -26,16 +50,12 @@ void top(char *dst) {
 
 void *mempcpy(void *restrict s1, const void *restrict s2, size_t n);
 
-void mempcpy14() {
+void mempcpy13() {
```

Comment:
Why are you renaming this to `memcpy13`? I see only two `memcpy` tests after this and they have index 15 and 16, which is very random, but at least follows 14 directly.

Reply:
I gave sensible names to all of these test functions.